config/presets/http.json

```diff
@@ -25,6 +25,9 @@
         {
           "@id": "urn:solid-server:default:Middleware"
         },
+        {
+          "@id": "urn:solid-server:default:PodHandler"
```

Comment:
Will the presence of this be a problem if people exclude `"files-scs:config/presets/pod-management.json"` from their config?

Reply:
Yes, but I'm not sure how we would get around that?

Reply:
One option could be to move the list of handlers to the "main" config files like we currently already do with the type of backend that is used, so it's easier to see?

Reply:
Since we can apply param values across files with Components.js, just moving this to the pod-management config should work:
```json
{
      "@id": "urn:solid-server:default:HttpHandler",
      "AllVoidCompositeHandler:_handlers": [
        {
          "@id": "urn:solid-server:default:PodHandler"
        }
      ]
    }
```

Reply:
But then we can't guarantee the order anymore right? And it's essential that the pod handler appears before the ldp handler in the array (since the last one is a catch-all for all requests that didn't get handled by anything else).

Reply:
Ah, I didn't know order is important. Ignore my comment in that case.